config/DynamicEvaluator/checker_phpstan.neon

```diff
@@ -6,3 +6,5 @@ parameters:
         analyse:
             - %currentWorkingDirectory%/src/Generated/*
             - %currentWorkingDirectory%/src/Orm/*
+            - %currentWorkingDirectory%/src/Pyz/Zed/*/Persistence/*Repository.php
```

Comment:
maybe better to ignore them like this             `- %currentWorkingDirectory%/src/*QueryContainer.php`
all files that ends *Repository.php & *QueryContainer.php . for now it's specific case for concrete project

Reply:
btw why have you ignored them?

Reply:
It is part of Spryker Php Syan configuration